_posts/en/meetings/2018-07-19-meeting.md

```diff
@@ -0,0 +1,304 @@
+bitcoin-seeder program should become part of the Bitcoin Core GitHub
+organization, whether the Bitcoin Core project should stop publishing
+release notes and binaries to Bitcoin.org, and information about the
+next CoreDev Tech meeting.
+
+Prior to the start of discussion, Wladimir van der Laan mentioned that
+the executables for Bitcoin Core 0.16.2 Release Candidate (RC) 2 are
+[available for testers][0.16.2rc2 bin].
+
+## High priority for review
+
+**Background:** each meeting, Bitcoin Core developers discuss which Pull
+Requests (PRs) the meeting participants think most need review in the
+upcoming week.  Some of these PRs are related to code that contributors
+especially want to see in the next release; others are PRs that are
+blocking further work or which require significant maintenance (rebasing)
+to keep in a pending state.  Any capable reviewers are encouraged to
+visit the project's list of [current high-priority
+PRs][].
+
+A special focus of this meeting was PRs that needed to be merged in the
+next few days if their features are to be included in upcoming version
+Bitcoin Core 0.17, so reviewers are also encouraged to check the list of
+PRs in the [0.17 milestone][].
+
+**Discussion ([log][log hipri]):** the following PRs were discussed:
+
+- [#9662][] - Add create wallet "disableprivatekeys" option: a sane mode
+  for watchonly-wallets.  Requested by Jonas Schnelli; Wladimir van der
+  Laan said it could probably be merged soon.
+
+- [#9502][] - [Qt] Add option to pause/resume block downloads.
+  Requested by Schnelli.
+
+- [#13697][] - Support output descriptors in scantxoutset.  Requested by
+  Pieter Wuille with support from Schnelli.  Discussed in more detail
+  [later](#osd) in the meeting.
+  
+  
+- [#13666][] - Always create signatures with Low R values.  Requested by
+  Wuille.
+
+- [#13426][] - [Bugfix] Add u8path and u8string to fix encoding issue
+  for Windows.  Requested by Chun Kuan Lee.
+
+- [#13712][] - Wallet: Fix non-determinism in ParseHDKeypath; avoid
+  using an uninitialized variable in path calculation.  Requested by
+  Andrew Chow.
+
+- [#8469][] - [POC] Introducing property-based testing to Core.
+  Requested to be removed from the 0.17 list by Schnelli, as it's not
+  ready.
+
+- [#13617][] - Require MacOS 10.10+.  Initially requested for removal
+  from the list, but added back after a comment by Cory Fields.
+
+**Conclusion:** there were over a dozen PRs tagged for 0.17 by the end
+of this segment of the meeting for a review deadline of Monday.
+
+## Exposing coin selection on RPC
+
+**Background:** Coin selection is the name used for how a wallet chooses
+which of a user's bitcoins to spend in a particular transaction.  The
+selected *Unspent Transaction Outputs (UTXOs)*, often simply called
+*coins,* become *inputs* in the transaction.  Developers have spent many
+hours in the past year improving Bitcoin Core's coin selection to be
```

Comment:
nit: many _hours_ undersells this a bit! Many _developer days_ or _weeks_ would be closer to the mark.